spec/lib/rwj_reporter_spec.rb

```diff
@@ -55,28 +50,15 @@
       it 'takes output_dir from settings if no param' do
         # covered by other tests, as print_report_for_dates is called w/o arg
       end
-      it 'each channel file is sorted by show number' do
-        rr.print_reports_for_dates
-        expect(channel1_output_array).to eq channel1_output_array.sort
-        expect(channel2_output_array).to eq channel2_output_array.sort
-      end
```

Comment:
If we are still having channel files, should we still be ensuring they are sorted by show number? Not clear to me why this test went away.